apollo-router/src/plugins/rhai.rs

```diff
@@ -516,15 +517,16 @@ macro_rules! gen_map_request {
                                 $callback.fn_name(),
                                 (shared_request.clone(),),
                             )
-                            .map_err(|err| err.to_string())
                     };
                     if let Err(error) = result {
-                        tracing::error!("map_request callback failed: {error}");
+                        let error_details = process_error(error);
+                        tracing::error!("map_request callback failed: {error_details}");
                         let mut guard = shared_request.lock().unwrap();
                         let request_opt = guard.take();
                         return failure_message(
                             request_opt.unwrap().context,
-                            format!("rhai execution error: '{}'", error),
+                            error_details,
+                            // format!("rhai execution error: '{}'", error),
```

Comment:
should we remove this line ?

Reply:
Yes, we should. Thank you. done: https://github.com/apollographql/router/pull/2097/commits/4646bf7df00a72e9d4c74d3fb3de82ac300219b5